contracts/GenArt721CoreV3_Engine.sol

```diff
@@ -0,0 +1,2137 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.17;
+
+// Created By: Art Blocks Inc.
+
+import "./interfaces/0.8.x/IRandomizerV2.sol";
+import "./interfaces/0.8.x/IAdminACLV0.sol";
```

Comment:
My thinking was that given the current ACL implementation is super simple (designed mainly to support the superuser use case only) we may as well add in support from this from the get-go as it will not functionally impact partners in how they interact with their contracts by default (we'll still ask for one initial wallet, multi-sig or cold wallet upon setup) but it does enable more flexibility/upgrade-ability to support partners' own business practices/contract management practices as needed, for those wanting to support more flexible behavior down the line. WDYT ser?

Reply:
Yeah, I think your logic there all makes sense 👍
One last thing I recently realized - did how we end up having `owner` return the AdminACL contract itself end up preventing automatic integration of contracts on OpenSea, etc.? For example, if we had the core contract's `owner` function return the adminACL's `superAdmin`, would integration with third party sites go smoother? (this is something I realized yesterday...) Too late for our flagship contract, but maybe something we could think of for the Engine contract.

Reply:
This is a great point ser – updated with https://github.com/ArtBlocks/artblocks-contracts/pull/361/commits/d638a55b09ffa412eb5cf64859e37065fd8efe70